cypress/integration/Popup.spec.ts

```diff
@@ -113,8 +113,7 @@ describe('Popup', () => {
           cy.findByRole('dialog', {name: 'Popup 2'}).should('be.visible');
         });
 
-        // TODO Skip for now until we have a systematic approach to fix this issue
-        it.skip('should close Popup 1', () => {
+        it('should close Popup 1', () => {
```

Comment:
I noticed this test was passing, so I removed the skip method here. Let me know if this should be reverted.

Reply:
This is passing in v5 as well, so I removed the `.skip` there. I think we ended up fixing the issue! 🎉